.github/workflows/l1-relayer-build.yml

```diff
@@ -0,0 +1,23 @@
+name: L1 Relayer - build
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}-l1
```

Comment:
No, it isn't. For some reason, "l2" workflow gets cancelled - no idea why, but maybe it's because it is the same group running from the same commit hash.